### PR TITLE
[SEMI-MODULAR] Makes paydays equal

### PR DESCRIPTION
**Pull request by softcerv**
## About The Pull Request
Removes the 3/4ths pay modifier on non-human races.

## How This Contributes To The Skyrat Roleplay Experience
Having pay based on race feels like a hold that doesn't fit in with skyrat, considering that a significant portion of he population of skyrat is made up of non-humans. Race based pay contributes little to nothing to roleplay because it doesn't factor in the economic factors that a character might have. Removing this allows us to make room for a better character based pay system in the future.

## Changelog

:cl:
balance: non-humans are now payed the same as humans.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/lizard.dm [hunk 1]
```diff
@@ -5,7 +5,7 @@
 	limbs_icon = 'modular_skyrat/master_files/icons/mob/species/lizard_parts_greyscale.dmi'
 	cultures = list(CULTURES_EXOTIC, CULTURES_LIZARD, CULTURES_HUMAN)
 	learnable_languages = list(/datum/language/common, /datum/language/draconic)
-	payday_modifier = 0.75
+	//SKYRAT EDIT - EQUAL PAY payday_modifier = 0.75
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/lizard.dm]: No need to comment it out here, it's modular.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Only thing that's missing I think are monkeys, but then again, they're not available at round-start so I doubt it matters much.

**Comment by softcerv** [comment]
> 
> 
> Only thing that's missing I think are monkeys, but then again, they're not available at round-start so I doubt it matters much.

aye, that and the 1.5x monkey pay modifier is more of a joke then anything else.

File: modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/lizard.dm (hunk 1 above)
Comment (softcerv) [reply modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/lizard.dm]: Got it, it's been deleted

**Review by GoldenAlpharex — APPROVED** [review]
It's good with me, I'll leave this up for a day since I know some people will want to discuss it.

**Comment by thestubborn** [comment]
we've had this argument before old man

https://github.com/Skyrat-SS13/Skyrat-tg/pull/5334

**Comment by softcerv** [comment]
> 
> 
> we've had this argument before old man
> 
> #5334

That was almost half a year ago.

**Comment by thestubborn** [comment]
> 
> 
> > we've had this argument before old man
> > #5334
> 
> That was almost half a year ago.

the pr still stands, someone tried this exact pr before, and it thankfully was closed

**Comment by softcerv** [comment]
> 
> 
> > > we've had this argument before old man
> > > #5334
> > 
> > 
> > That was almost half a year ago.
> 
> the pr still stands, someone tried this exact pr before, and it thankfully was closed

PRs are by no means concrete. This is something still worth discussing.

**Comment by thestubborn** [comment]
> 
> 
> > > > we've had this argument before old man
> > > > #5334
> > > 
> > > 
> > > That was almost half a year ago.
> > 
> > 
> > the pr still stands, someone tried this exact pr before, and it thankfully was closed
> 
> PRs can be over-ruled. This is something still worth discussing.

i disagree, this has been discussed, and vamp is still right, its one of the few flavour things that isn't clothing/purely cosmetic
and its far better and more interesting than this "no pay segregation" cumbia stuff

**Comment by softcerv** [comment]
> 
> 
> > > > > we've had this argument before old man
> > > > > #5334
> > > > 
> > > > 
> > > > That was almost half a year ago.
> > > 
> > > 
> > > the pr still stands, someone tried this exact pr before, and it thankfully was closed
> > 
> > 
> > PRs can be over-ruled. This is something still worth discussing.
> 
> i disagree, this has been discussed, and vamp is still right, its one of the few flavour things that isn't clothing/purely cosmetic
> and its far better and more interesting than this "no pay segregation" cumbia stuff

I am fine with pay segregation stuff, but this a horrible way of doing it that we inherited from TG and contributes little to nothing towards the roleplay experience of playing on Skyrat.

**Comment by thestubborn** [comment]
> 
> 
> > > > > > we've had this argument before old man
> > > > > > #5334
> > > > > 
> > > > > 
> > > > > That was almost half a year ago.
> > > > 
> > > > 
> > > > the pr still stands, someone tried this exact pr before, and it thankfully was closed
> > > 
> > > 
> > > PRs can be over-ruled. This is something still worth discussing.
> > 
> > 
> > i disagree, this has been discussed, and vamp is still right, its one of the few flavour things that isn't clothing/purely cosmetic
> > and its far better and more interesting than this "no pay segregation" cumbia stuff
> 
> I am fine with pay segregation stuff, but this a horrible way of doing it that we inherited from TG and contributes little to nothing towards the roleplay experience of playing on Skyrat.

how is it horrible? 0.75 is basically unnoticable
the roleplaying experience is subjective, it doesn't add anything ingame, but it does imply an institutional speciesism within a company that was founded by humans, which would happen
it just seems rather hugbox-y to me

**Comment by softcerv** [comment]
> 
> 
> > > > > > > we've had this argument before old man
> > > > > > > #5334
> > > > > > 
> > > > > > 
> > > > > > That was almost half a year ago.
> > > > > 
> > > > > 
> > > > > the pr still stands, someone tried this exact pr before, and it thankfully was closed
> > > > 
> > > > 
> > > > PRs can be over-ruled. This is something still worth discussing.
> > > 
> > > 
> > > i disagree, this has been discussed, and vamp is still right, its one of the few flavour things that isn't clothing/purely cosmetic
> > > and its far better and more interesting than this "no pay segregation" cumbia stuff
> > 
> > 
> > I am fine with pay segregation stuff, but this a horrible way of doing it that we inherited from TG and contributes little to nothing towards the roleplay experience of playing on Skyrat.
> 
> how is it horrible? 0.75 is basically unnoticable
> the roleplaying experience is subjective, it doesn't add anything ingame, but it does imply an institutional speciesism within a company that was founded by humans, which would happen
> it just seems rather hugbox-y to me

Having a system like this completely ignores the nuances that a company like NT might have when choosing how to pay people. I am completely fine with NT being racist, but this instance is ill-suited for Skyrat.

**Comment by GoldenAlpharex** [comment]
Please try to avoid arguing in circle, if you're planning on a big debate, take it to Discord. Also, if possible, try not to reply too many times in a row, it makes the comment section much less readable. ~~Yes I hate scrolling~~

**Comment by SpaceVampire** [comment]
Humans are chads.
The snowflake tax is inherently funny.
It also makes extra sense for some special needs races.
Like the Vox or Fire Skeletons.

**Review by ZephyrTFA — APPROVED** [review]
Racism in any form, even as a joke, is still racism. This is a good PR and the fact that people are trying to defend racism for zero reason, as getting paid less means nothing as money is worthless, is quite deplorable.

**Comment by thestubborn** [comment]
money SHOULD matter

and its not racism against things that are real, its racism against furry creatures

**Comment by Arctaisia** [comment]
Hi, NT Lore dev here.

NT doesn't care what the fuck someone's race is on the frontier, they're all expendable workers. All receive the same pay to keep them in the company all the same and reward them for the same work. NT doesn't want to spend extra on giving some species more than others, because then that may cause strikes or revolts from certain species.
This PR has my support because of this.

**Comment by thestubborn** [comment]
> 
> 
> Hi, NT Lore dev here.
> 
> NT doesn't care what the fuck someone's race is on the frontier, they're all expendable workers. All receive the same pay to keep them in the company all the same and reward them for the same work. NT doesn't want to spend extra on giving some species more than others, because then that may cause strikes or revolts from certain species.
> This PR has my support because of this.

no one's getting paid more, others are just getting paid less

**Comment by Arctaisia** [comment]
> > Hi, NT Lore dev here.
> > NT doesn't care what the fuck someone's race is on the frontier, they're all expendable workers. All receive the same pay to keep them in the company all the same and reward them for the same work. NT doesn't want to spend extra on giving some species more than others, because then that may cause strikes or revolts from certain species.
> > This PR has my support because of this.
> 
> no one's getting paid more, others are just getting paid less

That means people are being paid more than others, and being paid less means the exact same thing. This is a lore-friendly PR coming from one of the sole people working on this server's NT lore, and this should have been the default from the start.

Money should rather, be based on something like total play time for the job. NT wants to give it's best more money, as a means to make people strive for more work, to get more money, just like real life.

**Comment by thestubborn** [comment]
if we do implement this background money system...

what reason will anyone have to use anything but the highest tier?

**Comment by softcerv** [comment]
> 
> 
> if we do implement this background money system...
> 
> what reason will anyone have to use anything but the highest tier?

This is an RP server. Just like with quirks and loadouts, not everyone is going to pick the powergamer choices, the more agency we can give players about how their character works the better. I am not going to discuss this aspect any further on here because it is out of the scope of this PR.

**Comment by thestubborn** [comment]
> 
> 
> > if we do implement this background money system...
> > what reason will anyone have to use anything but the highest tier?
> 
> This is an RP server. Just like with quirks and loadouts, not everyone is going to pick the powergamer choices, the more agency we can give players about how their character works the better. I am not going to discuss this aspect any further on here because it is out of the scope of this PR.

a roleplay server doesn't mean people arent going to abuse it, we've removed less for less

**Comment by hugmeorelse** [comment]
IC Racism has always been something that was brought up in the Server Principle. It is meant to be a part of roleplay and the immersive experience, given that one does not go too far with it and end up being a dick towards people about it.

Also, differences in pay may not be motivated solely by racism itself but profit. If you can get away with paying certain employees less, a cut-throat corporation will do so and they can cite many legal reasons to do so like they do in real life. Immigrants, for instance that are looking to get a visa, get compensated less in turn for being sponsored. Some particular individuals are more desperate for a job and will accept a job for less. That's why corporations in real life illegally tell you that you cannot discuss pay with your co-workers. So, again, it may be motivated by racism but it's not done in sundown town fashion. It's a very complex topic that you often have people dedicating their academic life on this particular subject matter.

Anyhow, it's staying for now.

**Comment by Gandalf2k15** [comment]
As above

**Pull request closed without merging**